src/main/java/nl/tudelft/semgroup4/LoggerSetScreen.java

```diff
@@ -102,56 +61,48 @@ public LoggerSetScreen(ResourcesWrapper res, GameContainer container) throws Sli
-        typeFont.drawString(60, 330, cancel, Color.red);
+
+        renderText(60, 80);
 
         if (input.isMousePressed(Input.MOUSE_LEFT_BUTTON)) {
-            if (overCan.isMouseOver()) {
-                Game.LOGGER.log(LogSeverity.DEBUG, "LoggerSetScreen",
-                        "Player has cancelled setting the logger");
-                optionsState.toggleLoggerSet();
-            }
-            if (overCrit.isMouseOver()) {
-                Game.LOGGER.log(LogSeverity.DEBUG, "LoggerSetScreen",
-                        "Player has set the logger to critical");
-                Game.LOGGER.setSeverity(LogSeverity.CRITICAL);
-                optionsState.toggleLoggerSet();
-            }
-            if (overErr.isMouseOver()) {
-                Game.LOGGER.log(LogSeverity.DEBUG, "LoggerSetScreen",
-                        "Player has set the logger to error");
-                Game.LOGGER.setSeverity(LogSeverity.ERROR);
-                optionsState.toggleLoggerSet();
-            }
-            if (overWarn.isMouseOver()) {
-                Game.LOGGER.log(LogSeverity.DEBUG, "LoggerSetScreen",
-                        "Player has set the logger to warning");
-                Game.LOGGER.setSeverity(LogSeverity.WARNING);
-                optionsState.toggleLoggerSet();
-            }
-            if (overDeb.isMouseOver()) {
-                Game.LOGGER.log(LogSeverity.DEBUG, "LoggerSetScreen",
-                        "Player has set the logger to debug");
-                Game.LOGGER.setSeverity(LogSeverity.DEBUG);
-                optionsState.toggleLoggerSet();
+            for (int i = 0; i < areas.length; i++) {
+                if (areas[i].isMouseOver()) {
+                    Game.LOGGER.log(LogSeverity.DEBUG, "LoggerSetScreen",
+                            "Player has set the logger to " + text[i]);
+                    Game.LOGGER.setSeverity(severities[i]);
+                    optionsState.toggleLoggerSet();
+                }
             }
+        }
+    }
 
-            if (overVerb.isMouseOver()) {
-                Game.LOGGER.log(LogSeverity.DEBUG, "LoggerSetScreen",
-                        "Player has set the logger to verbose");
-                Game.LOGGER.setSeverity(LogSeverity.VERBOSE);
-                optionsState.toggleLoggerSet();
-            }
+    /**
+     * Renders all the buttons.
+     *
+     * @param renderCoordX
+     *                  the x coordinate to use
+     * @param renderCoordY
+     *                  the first y coordinate to use
+     */
+    public void renderText(float renderCoordX, float renderCoordY) {
+        for (int i = 0; i < text.length; i++) {
+            typeFont.drawString(renderCoordX,
+                    renderCoordY + i * 50,
+                    text[i],
+                    controller.decideColor(severities[i]));
         }
     }
 
+    /**
+     * Get all mouse over areas (only used for testing).
+     * @param overArea
```

Comment:
Oh whoops, I changed what I wanted that method to do but forgot to change the javadoc. Will change!